### PR TITLE
chores(javascript): cleanup NPM and Webpack

### PR DESCRIPTION
**Pull request by cweider**
Fixes issues:
- Dependency resolution error c/o `webpack-cli` had prevented `npm install`
- Current packaged Javascript hasn't matched compiled Webpack output since upgrade to version 5 (8256590)
- `npm audit` reported several vulnerable packages

### CODE REVIEW

File: cl/assets/static-global/js/react/vendor.js.map [hunk 1]
```diff
@@ -1 +0,0 @@
-{"version":3,"file":"vendor.js","sources":["webpack:///vendor.js"],"mappings":"AAAA;;;;;AA8ZA;AAyoCA;;;;AA8sKA;;;;;AAsvaA;;;;;;;AAy/FA;;;;;;;;AAkiRA;;;;;;;;AAmCA;;;;;;;;AAoTA;;;;;;;;AA28BA;;;;;;;;AA0YA;;;;;;;;AAg7DA;;;;;;;;AAkCA","sourceRoot":""}
```
Comment (cweider) [inline cl/assets/static-global/js/react/vendor.js.map:1]: I’m not sure why `sources` is only `["webpack:///vendor.js"]` (or how to generate _this_ output). The good news is that it means this source map is essentially meaningless – and harmless to remove.

File: cl/webpack.config.js [hunk 2]
```diff
@@ -23,6 +24,13 @@ module.exports = {
       name: 'vendor',
       filename: '[name].js',
     },
+    minimize: true,
+    minimizer: [new TerserPlugin({
+        extractComments: false,
+        terserOptions: {
+          compress: true,
+        },
+      })],
```
Comment (cweider) [inline cl/webpack.config.js:33]: I don’t love this non-default plugin, but this is evidently how you preserve comments. Moving to `LICENSE.txt` (the default imposed by Webpack 5) is something to consider, but, on balance, the added complication of this plugin is worth removing urgency of reconsidering our source attribution approach!

File: cl/assets/static-global/js/react/main.js.map [hunk 3]
```diff
@@ -1 +0,0 @@
-{"version":3,"file":"main.js","sources":["webpack:///main.js"],"mappings":"AAAA","sourceRoot":""}
```
Comment (cweider) [inline cl/assets/static-global/js/react/main.js.map:1]: cf. `vendor.js.map`

**Comment by mlissner** [comment]
LGTM, thanks.

**Pull request merged**